src/components/SearchBar/SearchBar.tsx

```diff
@@ -10,9 +10,9 @@ function SearchBar({
   placeholder,
   searchLocation,
 }: SearchBarProps): JSX.Element {
-  const handleSubmit = async (event: FormEvent) => {
+  const handleSubmit = (event: FormEvent) => {
     event.preventDefault();
-    await search(searchLocation);
+    console.log(searchLocation); //Add a search function later
```

Comment:
If you want to add it later you might as well just give this component a prop, that accepts a function and is called during submitting :)